.pipelines/int-release.yml

```diff
@@ -10,6 +10,13 @@ parameters:
   type: string
   default: latest
 
+resources:
```

Comment:
Uses `AROGIT` service connection for readonly access to msazure organization to pull the config from `AzureRedHatOpenShift/RP-Config` repository.